src/test/mir-opt/uninhabited_enum_branching.rs

```diff
@@ -14,6 +14,8 @@ enum Test2 {
     E = 5,
 }
 
+// EMIT_MIR rustc.main.UninhabitedEnumBranching.diff
+// EMIT_MIR rustc.main.SimplifyCfg-after-uninhabited-enum-branching.after.mir
```

Comment:
I've seen this pattern a few times. I wonder if we should have a diff mode for "before Optimization X to after optimization Y"

Reply:
Maybe we should always use `.diff`.

Reply:
What I mean is to diff across multiple optimizations instead of just a single one.